src/features/vote/CommitPhase.tsx

```diff
@@ -145,30 +144,25 @@ const CommitPhase: FC<Props> = ({
             commitVotes(vc, fd)
               .then((tx) => {
                 // console.log("tx", tx);
-                setModalState("pending");
                 setSubmitErrorMessage("");
-
+                close();
+                reset();
```

Comment:
so my understanding is instead of leaving modal open after commit, it gets closed and notify js will handle the rest?

Reply:
Yeah exactly.